reacthomeworkweek2/App.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+class Container extends React.Component {
```

Comment:
you have fit your program in a one Component.
Changing that now will be changing most of your work, so that's fine, I will be expecting more components in the next homework for more legibility, and ise of practice. 🔥